apps/etterlatte-opplysninger-fra-soeknad/src/main/kotlin/opplysningerfrasoknad/Opplysningsuthenter.kt

```diff
@@ -310,6 +301,65 @@ class Opplysningsuthenter {
     }
 }
 
+private object OmstillingsstoenadUthenter {
+    fun lagOpplysningsListe(jsonNode: JsonNode): List<Grunnlagsopplysning<out Any?>> {
+        val barnepensjonssoknad = objectMapper.treeToValue<Omstillingsstoenad>(jsonNode)
+
+        return listOfNotNull(
+            avdoedOpplysning(barnepensjonssoknad)
+            // soeker(barnepensjonssoknad, Opplysningstype.SOEKER_SOEKNAD_V1),
+            // innsender(barnepensjonssoknad, Opplysningstype.INNSENDER_SOEKNAD_V1),
+            // utbetalingsinformasjon(barnepensjonssoknad, Opplysningstype.UTBETALINGSINFORMASJON_V1),
+            // søkerUtenlandsopphold,
+            // utenlandsoppholdAvdød,
+            // samtykke(barnepensjonssoknad, Opplysningstype.SAMTYKKE),
+            // spraak(barnepensjonssoknad, Opplysningstype.SPRAAK),
+            // soeknadMottattDato(barnepensjonssoknad, Opplysningstype.SOEKNAD_MOTTATT_DATO),
+            // soeknadsType(barnepensjonssoknad, Opplysningstype.SOEKNADSTYPE_V1),
+            // personGalleri(barnepensjonssoknad),
+            // gjenlevendeForelder(barnepensjonssoknad, Opplysningstype.GJENLEVENDE_FORELDER_SOEKNAD_V1),
+        )
+    }
+
+    private fun kilde(soknad: Omstillingsstoenad): Grunnlagsopplysning.Kilde {
+        return Grunnlagsopplysning.Privatperson(
+            soknad.innsender.foedselsnummer.svar.value,
+            soknad.mottattDato.toInstant(ZoneOffset.UTC)
+        )
+    }
+
+    private fun avdoedOpplysning(soknad: Omstillingsstoenad): Grunnlagsopplysning<out Any?> {
```

Comment:
Liker ikkje heilt at vi bruker any når vi kan unngå det, men ser ut som den er introdusert her for lengst.

Reply:
Har kopiert litt av det som var brukt fra før ja :) Men denne er eksplisitt nok til å kunne droppe any. Fikser 👍